pkg/webhook/statefulset/validating/statefulset_validation.go

```diff
@@ -44,15 +43,16 @@ func validatePodManagementPolicy(spec *appsv1beta1.StatefulSetSpec, fldPath *fie
 
 func validateReserveOrdinals(spec *appsv1beta1.StatefulSetSpec, fldPath *field.Path) field.ErrorList {
 	var allErrs field.ErrorList
-
-	if spec.ReserveOrdinals != nil {
-		orders := sets.NewInt(spec.ReserveOrdinals...)
-		if orders.Len() != len(spec.ReserveOrdinals) {
-			allErrs = append(allErrs, field.Invalid(fldPath.Root(), spec.ReserveOrdinals, "reserveOrdinals contains duplicated items"))
-		}
-		for _, i := range spec.ReserveOrdinals {
-			if i < 0 {
-				allErrs = append(allErrs, field.Invalid(fldPath.Root(), spec.ReserveOrdinals, fmt.Sprintf("reserveOrdinals contains %d which must be order >= 0", i)))
```

Comment:
plz keep the duplication checking

Reply:
I also think it should be more strictly judged, e.g. refer to GetIntSet